docs/docs/APIGuide/Layers/Recurrent-Layers.md

```diff
@@ -175,6 +175,9 @@ With RecurrentDecoder, inputsize and hiddensize of the cell must be the same.
 
 Different types of rnn cells can be added using add() function.
 
+RecurrentDecoder supports returning state and cell of its rnn cells at last time step by using getState.
+If contained cell is MultiCell, use getStates/setStates instead.
```

Comment:
Can we unify the interface?

Reply:
get/setStates use an array of state while get/setState use a single state. To unify the interface, one thought is using interface of get/setStates, in this way for non-multirnn cell, we have to wrap a state to an array before pass it to get/setStates. I am not sure if it's straight forward enough for non-multirnn cell.